src/App.tsx

```diff
@@ -1,61 +1,148 @@
+import React, {
+  ChangeEvent, FormEvent, useState,
+} from 'react';
+import fetchedUsers from './api/users';
+import fetchedTodos from './api/todos';
+import { User } from './types';
+import { TodoList } from './components/TodoList';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const todosWithAssociatedUsers = fetchedTodos.map(todo => ({
+  ...todo,
+  user: fetchedUsers.find(user => user.id === todo.userId),
+}));
 
 export const App = () => {
+  const [todoItems, setTodoItems] = useState(todosWithAssociatedUsers);
+  const [formValues, setFormValues] = useState({ title: '', user: '' });
+  const [isValidTitle, setIsValidTitle] = useState(false);
+  const [isValidUser, setIsValidUser] = useState(false);
+  const [displayTitleError, setDisplayTitleError] = useState(false);
+  const [displayUserError, setDisplayUserError] = useState(false);
```

Comment:
It kind of looks like two pieces of state for the same thing. Could you simplify it?